lib/Distribution/Opensuse/Leap.pm

```diff
@@ -0,0 +1,23 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: The class represents Leap distribution and provides access to its
+# features.
+# Follows the "Factory first" rule. So that the feature first appears in
+# Tumbleweed distribution, and only if it behaves different in Leap then it
+# should be overriden here.
+
+# Maintainer: Oleksandr Orlov <oorlov@suse.de>
+
+package Distribution::Opensuse::Leap;
+use strict;
+use warnings FATAL => 'all';
+use parent 'Distribution::Opensuse::Tumbleweed';
```

Comment:
Actually Leap is derived from SLE *and* Tumbleweed depending on the component so encoding it as simple as that is not true. For the installer however you can expect it to be similar to SLE regarding functionality but us the openSUSE theming which should only be visible in needles.

Reply:
So, here I would go with the existing solution as for now. As for the partitioning lvm it completes all the needs. In case if sometime we'll need to extend hierarchy, then it could be done. 

One smart guy just recently explained me "Never Add Functionality Early" principle;)